### DISCUSSION TITLE
Error when trying to load FASTA: src/diann.cpp: 19102: Cannot load FASTA

### DISCUSSION THREAD

**Discussion by nbender2** [vdemichev/DiaNN#1007] [General]
I am currently trying to switch from Windows to a Linux-based server as the amount of data is getting out of hand. While trying to set up the commandline i ran into the following error:
```ERROR: src/diann.cpp: 19102: Cannot load FASTA```
The Fasta-File is identical to the one I used on Windows and I have checked everything regarding permissions. At the moment, I am completely out of ideas and would appreciate any kind of input. For better comparison I have provided my commandline code below:

```./diann-1.8.1 --dir /home/ubuntu/Omics/Data/mzml --lib "" --threads 8 --verbose 1 --out /home/ubuntu/Omics/DIA-NN/benchmark --qvalue 0.01 --out-lib /home/ubuntu/Omics/DIA-NN/benchmark --gen-spec-lib --predictor --fasta /home/ubuntu/Omics/Data/REF_EColi_K12_UPS1_combined.fasta --fasta search --min-fr-mz 50 --max-fr-mz 1500 --met-excision --cut K*,R* --missed-cleavages 2 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 350 --max-pr-mz 950 --min-pr-charge 1 --max-pr-charge 4 --unimod4 --var-mods 3 --reanalyse --relaxed-prot-inf --smart-profiling  --fasta-search```

```
3 files will be processed
[0:00] Loading FASTA /home/ubuntu/Omics/Data/REF_EColi_K12_UPS1_combined.fasta
[0:00] Loading FASTA search
cannot read the file
ERROR: src/diann.cpp: 19102: Cannot load FASTA
```
Best regards

**Comment by vdemichev** [comment]
--fasta search - this is the typo, should be --fasta-search.
General comment: please create a predicted library in a separate step, i.e. first create and then analyse with it.
